cmake/modules/contrib/TF_TVMDSOOP.cmake

```diff
@@ -0,0 +1,48 @@
+
+if(NOT USE_TF_TVMDSOOP STREQUAL "OFF")
+
+  # If want build this directly comment out below lines.
+  # if ("${TVM_HOME}" STREQUAL "")
+  #   message(FATAL_ERROR "TVM_HOME is not defined")
+  # else()
+  #  message("Use TVM_HOME=\"${TVM_HOME}\"")
+  #endif()
+  # include_directories(${TVM_HOME}/include)
+  # include_directories(${TVM_HOME}/3rdparty/dlpack/include)
+  # include_directories(${TVM_HOME}/3rdparty/dmlc-core/include)
+  # set(TFTVM_LINK_FLAGS  -ltvm_runtime -L${TVM_HOME}/build)
```

Comment:
what's these for?

Reply:
This was used for tftvm project when the code base is not in tvm and we have remove the comment now.